plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestingMySqlServer.java

```diff
@@ -30,7 +30,7 @@
 public class TestingMySqlServer
         implements AutoCloseable
 {
-    public static final String DEFAULT_IMAGE = "mysql:8.0.12";
+    public static final String DEFAULT_IMAGE = "mysql:8.0.29";
```

Comment:
Please use `mysql:8.0.29-oracle` since that one supports both AMD64 and ARM64, the latter of which is needed for running on M1 Mac.

Also update all the other usages of the image. See https://github.com/trinodb/trino/pull/12692/commits/c3b0a61e99a93ddf3e35ca55e0a16088efa1750d for a list. (we could consider adding a constant for this in `trino-testing`)

Reply:
Updated all the usage of the image. Thanks!  Try to think how to add a constant. will update later.

Reply:
I think we can leave the mysql container image constant in the diff files, because the `trino-session-property-managers` and `trino-verifier` are NOT depended on `trino-testing` project, will we need keep the dependence simple? And will we need all the mysql images the same, even though they are NOT test the Mysql? Thanks!